source/common/access_log/access_log_formatter.h

```diff
@@ -23,6 +25,12 @@ class AccessLogFormatParser {
 private:
   static void parseCommand(const std::string& token, const size_t start, std::string& main_header,
                            std::string& alternative_header, absl::optional<size_t>& max_length);
+
+  // the indexes of where the parameters for each directive is expected to begin - generally the
+  // length of the directive + 1 for the opening '('
+  static const size_t ReqParamStart{4};
```

Comment:
It's still unclear where these numbers come from. Can you actually put in an initializer such as `"FOO(".size()` here?